pontoon/base/signals.py

```diff
@@ -17,3 +25,78 @@ def project_locale_added(sender, **kwargs):
         project = project_locale.project
         project.has_changed = True
         project.save(update_fields=['has_changed'])
+
+
+@receiver(pre_save, sender=Locale)
+def create_locale_translators_group(sender, **kwargs):
+    """
+    Creates respective translators group in django's permission system.
+    Managers should be able to translate and manage locale.
+    """
+    instance = kwargs['instance']
+
+    if kwargs['raw'] or instance.translators_group is not None:
+        return
+    try:
+        locale_ct = ContentType.objects.get(app_label='base', model='locale')
+        locale_group, _ = Group.objects.get_or_create(name='{} translators'.format(instance.code.upper()))
+        can_translate = Permission.objects.get(content_type=locale_ct, codename='can_translate_locale')
+        locale_group.permissions.add(can_translate)
+        instance.translators_group = locale_group
+    except ObjectDoesNotExist:
+        errors.send_exception(*sys.exc_info())
```

Comment:
Huh, I misread https://github.com/MindscapeHQ/raygun4py/blob/master/python2/raygun4py/raygunprovider.py#L74 and missed that it takes the traceback from exc_info when an exception is passed in. Either way, since they already call exc_info to handle it for you you don't need to call it yourself, but it's equivalent either way, so whatevs. :D